misc/docker/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN git clone https://github.com/nitlang/nit.git /nit \
 	&& make \
 	&& . misc/nit_env.sh install \
 	# Clean and reduce size
-	&& strip c_src/nitc bin/nit* \
+	&& { strip c_src/nitc bin/nit* ; true ; } \
```

Comment:
Are you sure this won’t stop before processing all the compiled files?

PS: Why there is a space before `;`? Also, I think `{ strip c_src/nitc bin/nit* || true; }` would be clearer (and not dependent on `set +e`).